GitUI/Properties/Resources.Designer.cs

```diff
@@ -1751,6 +1815,26 @@ internal static System.Drawing.Bitmap IconBranchCheckout
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized resource of type System.Drawing.Bitmap.
+        /// </summary>
+        internal static System.Drawing.Bitmap RemoteBranch {
+            get {
+                object obj = ResourceManager.GetObject("RemoteBranch", resourceCulture);
+                return ((System.Drawing.Bitmap)(obj));
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized resource of type System.Drawing.Bitmap.
+        /// </summary>
+        internal static System.Drawing.Bitmap RemoteRepo {
+            get {
+                object obj = ResourceManager.GetObject("RemoteRepo", resourceCulture);
+                return ((System.Drawing.Bitmap)(obj));
```

Comment:
Please have all new images recorded in [_Icon Sources.txt](https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/Icons/originals/_Icon%20Sources.txt)

Reply:
TBH, I don't know where did the original author (@bergerjac, inactive for a couple of years now) get the images.

Reply:
Check if you can find comparable images here (https://vsicons.azurewebsites.net/) or here (https://www.microsoft.com/en-us/download/details.aspx?id=35825) - same content just different form.
These come under [MS licence](http://download.microsoft.com/download/0/6/0/0607D8EA-9BB7-440B-A36A-A24EB8C9C67E/Visual%20Studio%202017%20Image%20Library%20EULA.RTF) are free to use and redistribute

Reply:
@EbenZhang icons were pulled from VS 2010 or 2012 icon folders as @RussKie mentioned.

Reply:
Please check if icons from VS2017 are more colorful and not so depressing

Reply:
If this pull request was a batch of delicious cookies, the icons are a crumb, and tiny compared to the whole.

Reply:
I'd like to see them more as a cherry on the top of the cake

Reply:
very nice and exciting to see u here